components/automate-ui/src/app/pages/api-token/details/api-token-details.component.html

```diff
@@ -29,20 +29,30 @@
         </chef-tab-selector>
       </chef-page-header>
     <section class="page-body">
-      <form [formGroup]="updateNameForm">
+      <form [formGroup]="updateForm">
         <chef-form-field>
           <label>Name *</label>
-          <input chefInput formControlName="name" type="text" (keyup)="handleNameChange()">
+          <input chefInput formControlName="name" type="text">
+          <chef-error *ngIf="updateForm.get('name').hasError('required') && updateForm.get('name').dirty">
+            Name is required.<!-- drop period? -->
+          </chef-error>
+          <!-- error for NON_BLANK validator? -->
+        </chef-form-field>
+        <chef-form-field>
+          <label>Status *</label>
+          <chef-phat-radio ngDefaultControl formControlName="status">
+            <chef-option value="active">Active</chef-option>
+            <chef-option value="inactive">Inactive</chef-option>
+          </chef-phat-radio>
         </chef-form-field>
       </form>
-      <chef-button primary [disabled]="!updateNameForm.valid || disableSave || saveInProgress" (click)="saveNameChange()">
+      <div id="save">
+      <chef-button primary [disabled]="!updateForm.valid || !updateForm.dirty || saveInProgress" (click)="saveChange()">
```

Comment:
Yes! This is the way to go! I've updated the backing code to properly support this.